client/my-sites/posts/style.scss

```diff
@@ -619,6 +623,7 @@
 	flex-grow: 1;
 	margin: 0 0 24px 24px;
 	display: none;
+	max-width: 400px;
```

Comment:
Right. As for where did the number `400px` come from, I just thought it looked best. It gave enough space for the drafts while still keeping it smaller than the post list.